### DISCUSSION TITLE
customization of statusline for simplified override module

### DISCUSSION THREAD

**Discussion by benzwt** [NvChad/NvChad#2299] [Q&A] [answered]
The latest nvchad uses simplified override modules for statusline https://github.com/NvChad/ui/pull/152

The new way of overriding the module was documented at https://nvchad.com/docs/config/nvchad_ui#statusline___tabufline

I'm not intend to make a big change on the modules, but rather making minor change
based on the nvchad.statusline.
I followed the guideline of in https://github.com/NvChad/NvChad/discussions/2209; to write a custom.statusline.lua and put
> modules[2] = somefunction()" 

to ui.statusline.  

However, the lua failed at 
> require "nvchad.statusline.default" 
 
in my custom statusline.lua
Was there any crucial step I have missed ?

**Comment by siduck** [comment]
require "nvchad.statusline.default"

that isnt even required. please check the docs well, then  you'll know how the overriding works. dont follow that thread

**Reply by benzwt** [reply, marked as answer]
Thank I got it ! 
The solution is in https://github.com/NvChad/NvChad/discussions/2246
